README.md

```diff
@@ -25,7 +25,14 @@ steps.forEach(step => {
 
 To solve an equation:
 ```js
-const steps = mathsteps.solveEquation('2x + 2x + x + x');
+const steps = mathsteps.solveEquation('2x + 3x = 35');
+
+steps.forEach(step => {
+    console.log("before change: " + step.oldEquation.print());  // e.g. before change: 2x + 3x = 35
```

Comment:
I think it can take 100 characters with ease without bringing in scroll bars. And a few more characters, from what I can see. Not more. And my code has only one line which has 100 characters and that's the max